outbound/parser_test_data.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "FirstName": "Eric",
+    "LastName": "Rose",
+    "DOB": "1989-09-07",
+    "EmailAddress": "eric.rose97@coruscant.space",
+    "Address": "954 Hanson Turnpike, Ericafort, VA 19735",
+    "SSN": "710-37-0415",
+    "PhoneNumber": "535.250.0117",
+    "InsuranceID": "INS-8990-8170",
+    "InsuranceCarrier": "Aetna",
+    "Symptoms": "Abdominal Pain; Runny Nose",
+    "Diagnosis": "COVID-19"
+  }
+]
```

Comment:
Ideally, adding inbound, outbound, and processed to the git ignore would keep this from being committed.

Thankfully this is all fake test data, otherwise this would be considered a critical security breach of data.